R/uvhydrograph-data.R

```diff
@@ -20,19 +21,23 @@ parseUVData <- function(data, plotName, month) {
     meas_Q <- subsetByMonth(getFieldVisitMeasurementsQPoints(data), month)  
     
     primary_lbl <- getUvLabel(data, "primarySeries")
+    dvmin_lbl <- getUvLabel(data, "derivedSeriesMin")
```

Comment:
@lindsaycarr here is where I tried to use the getUvLabel
but was getting the "$ operator is invalid for atomic vectors" coming from the isEmptyVar function on the variable incoming, which was just "Temperature, water degC"